workflow-templates/hqs-ci-test-pure-python.yml

```diff
@@ -0,0 +1,28 @@
+name: HQS CI tests for pure python repos/folders
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [$default-branch]
+
+jobs:
+  linting:
+    uses: HQSquantumsimulations/reusable_workflows/.github/workflows/reusable_linting_pure_python.yml@main
+    with: 
+      python_folder: "NAME_OF_TOP_LEVEL_PYTHON_FOLDER"
+      linting_folder: "NAME_OF_PYTHON_FOLDER_TO_LINT"
+      # Run tests also on windows runners
+      windows: false
+      # Run tests also on macos runners
+      macos: true
+
+  unittests:
+    uses: HQSquantumsimulations/reusable_workflows/.github/workflows/reusable_tests_pure_python.yml@main
+    with:
+      python_folder: "NAME_OF_TOP_LEVEL_PYTHON_FOLDER"
+      test_code_coverage: true
+      # Run tests also on windows runners
+      windows: false
+      # Run tests also on macos runners
+      macos: true
+      # Test code coverage of rust core is over 90 percent
```

Comment:
This comment is from the Rust workflow.
Writing something similar just over line 23 is useful. For Python it's 80%.

Reply:
Good point, thank you!